### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Julienraptor01** [fxmorin/memoryLeakFix#5]
Version 0.3.2 make me crash while 0.3.1-forceload doesn't

[latest.log 0.3.2](https://github.com/fxmorin/memoryLeakFix/files/8791748/latest.log)

[latest.log 0.3.1-forceload](https://github.com/fxmorin/memoryLeakFix/files/8791745/latest.log)

**Comment by FxMorin** [comment]
@altrisi I'm not quite sure what the issue is here

**Comment by craftish37** [comment]
Well mates I have a crash-report, dunno if it's the same issue but same result (:
[crash-2022-05-28_23.42.57-client.txt](https://github.com/fxmorin/memoryLeakFix/files/8792212/crash-2022-05-28_23.42.57-client.txt)

**Comment by NaiNonTH** [comment]
This was caused by `java.lang.StackOverflowError` (Stack Overflow).
[2022-05-29-3.log](https://github.com/fxmorin/memoryLeakFix/files/8792647/2022-05-29-3.log)

```
...
[11:39:28] [Render thread/ERROR]: Unhandled game exception
java.lang.RuntimeException: Mixin transformation of team.creative.creativecore.common.config.holder.CreativeConfigRegistry failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:418) ~[fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:322) ~[fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.6.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at team.creative.creativecore.common.config.event.ConfigEventHandler.loadClientFields(ConfigEventHandler.java:278) ~[CreativeCore_FABRIC_v2.6.10_mc1.18.2.jar:?]
	at team.creative.creativecore.common.config.event.ConfigEventHandler.load(ConfigEventHandler.java:307) ~[CreativeCore_FABRIC_v2.6.10_mc1.18.2.jar:?]
	at team.creative.creativecore.common.config.event.ConfigEventHandler$ClientHandler$Handler.onClientStarted(ConfigEventHandler.java:69) ~[CreativeCore_FABRIC_v2.6.10_mc1.18.2.jar:?]
	at net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientLifecycleEvents.lambda$static$0(ClientLifecycleEvents.java:38) ~[fabric-lifecycle-events-v1-2.0.4+d8d7804a60-d80747583937e2d1.jar:?]
	at net.minecraft.class_310.handler$bfc000$onStart(class_310.java:4483) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:716) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:236) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:460) [fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.6.jar:?]
Caused by: java.lang.StackOverflowError
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1421) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.ClassInfo.findSuperClass(ClassInfo.java:1426) ~[sponge-mixin-0.11.4+mixin.0.8.5.jar:0.11.4+mixin.0.8.5]
.
.
. (and much more replicated error source)
[11:39:28] [Render thread/INFO]: Stopping!
[11:39:28] [CullThread/INFO]: [STDOUT]: Shutting down culling task!
[11:39:28] [Render thread/INFO]: Better Taskbar is available
[11:39:28] [Render thread/WARN]: Configuration conflict: there is more than one oshi.architecture.properties file on the classpath
[11:39:29] [Render thread/ERROR]: Shutdown failure!
java.util.ConcurrentModificationException: null
	at java.util.HashMap.forEach(HashMap.java:1424) ~[?:?]
	at net.minecraft.class_1060.close(class_1060.java:173) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.close(class_310.java:1048) [client-intermediary.jar:?]
	at net.minecraft.class_310.method_1490(class_310.java:1026) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:252) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:460) [fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.6.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.6.jar:?]
```

**Comment by Funtimes909** [comment]
Adding "-ForceLoad" to the end of the .jar file name works for me (If that doesn't work try without the dash)

**Comment by FxMorin** [comment]
wtf, how? That's not how that works

**Comment by FxMorin** [comment]
Can everyone here please check if your crash issue was fixed in v0.3.3
If it was not, please make a separate bug report so I can easily find the issue. Thanks

**Comment by Julienraptor01** [comment]
The bug is fixed for me
